### PR TITLE
Added QRCode and link to configure android application

### LINKED ISSUE #2277: Prefill android app with user & server informations through a link
Inspiration from https://github.com/nextcloud/android/issues/248 : Prefill the Android app with user and server informations.

See https://github.com/wallabag/android-app/issues/290 too

### PR DESCRIPTION
**Pull request by nicosomb**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Documentation | no
| Translation   | yes
| Fixed tickets | #2277, #2448
| License       | MIT

Fix #2277, #2448 
We use the magic URL defined here https://github.com/wallabag/android-app/pull/300

On mobile view, we have a button:

On desktop view, a QR Code: 

Todo: 

- [ ] documentation
- [x] translations
- [ ] some love for the display :) Help needed! 
- [x] add this part in baggy theme

### CODE REVIEW

File: app/Resources/static/themes/_global/js/tools.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
 const $ = require('jquery');
 
+var jrQrcode = require('jr-qrcode');
```
Comment (tcitworld) [inline app/Resources/static/themes/_global/js/tools.js]: No `var`, use `const` instead.
But you better go with es6 imports like this : `import jrQrcode from 'jr-qrcode`

File: src/Wallabag/CoreBundle/Resources/views/themes/material/Config/index.html.twig [hunk 2]
```diff
@@ -71,6 +71,18 @@
...
                             </div>
 
+                            <div class="row">
+                                <div class="input-field col s12">
+                                    <h5>Configure your Android application</h5>
+                                    <a href="wallabag://{{ app.user.username }}@{{ wallabag_url }}" class="waves-effect waves-light btn hide-on-large-only">Touch here to prefill your Android application</a>
+                                    <img id="androidQrcode" class="hide-on-med-and-down" />
+                                </div>
+                                <script>
+                                    var imgBase64 = jrQrcode.getQrBase64('wallabag://{{ app.user.username }}@{{ wallabag_url }}');
```
Comment (tcitworld) [inline src/Wallabag/CoreBundle/Resources/views/themes/material/Config/index.html.twig]: Use `const`

File: src/Wallabag/CoreBundle/Resources/views/themes/baggy/Config/index.html.twig [hunk 3]
```diff
@@ -54,6 +54,16 @@
                     <a href="https://getpocket.com/developer/docs/authentication">https://getpocket.com/developer/docs/authentication</a>
                 </p>
             </div>
+
+            <div class="row">
+                    <h3>{{ 'config.form_settings.android_configuration'|trans }}</h3>
```
Comment (j0k3r) [inline src/Wallabag/CoreBundle/Resources/views/themes/baggy/Config/index.html.twig]: Too much spaces here

File: src/Wallabag/CoreBundle/Resources/views/themes/baggy/Config/index.html.twig [hunk 4]
```diff
@@ -54,6 +54,16 @@
...
+                    <a href="wallabag://{{ app.user.username }}@{{ wallabag_url }}" >Touch here to prefill your Android application</a>
+                    <img id="androidQrcode" />
+                    <script>
+                        const imgBase64 = jrQrcode.getQrBase64('wallabag://{{ app.user.username }}@{{ wallabag_url }}');
+                        document.getElementById('androidQrcode').src=imgBase64;
```
Comment (j0k3r) [inline src/Wallabag/CoreBundle/Resources/views/themes/baggy/Config/index.html.twig]: Spaces needed between `=`

File: src/Wallabag/CoreBundle/Resources/views/themes/material/Config/index.html.twig [hunk 5]
```diff
@@ -71,6 +71,18 @@
...
 
+                            <div class="row">
+                                <div class="input-field col s12">
+                                    <h5>{{ 'config.form_settings.android_configuration'|trans }}</h5>
+                                    <a href="wallabag://{{ app.user.username }}@{{ wallabag_url }}" class="waves-effect waves-light btn hide-on-large-only">Touch here to prefill your Android application</a>
+                                    <img id="androidQrcode" class="hide-on-med-and-down" />
+                                </div>
+                                <script>
+                                    const imgBase64 = jrQrcode.getQrBase64('wallabag://{{ app.user.username }}@{{ wallabag_url }}');
+                                    document.getElementById('androidQrcode').src=imgBase64;
```
Comment (j0k3r) [inline src/Wallabag/CoreBundle/Resources/views/themes/material/Config/index.html.twig]: Here too

**Comment by nicosomb** [comment]
@tcitworld is that OK for the display?

**Comment by tcitworld** [comment]
:+1:

**Pull request merged**